### ISSUE TITLE
New Mob Suggestion

### ISSUE THREAD

**Issue by NathanCPlaster** [AlexModGuy/Ice-and-Fire-Suggestions#1320]
Mob to add: Goblins, nests should spawn semi frequently and they will venture out of the nest.  The nest is cave like in structure.  The goblins will attempt to loot chests as well as attack the player and farm animals.  The nest will have a maximum spawn of 15 and spawn new ones a day later should they die.  Upon clearing the nest out no new goblins will spawn and there is a loot room of things they have scavenged.

**Comment by CTH999** [comment]
Goblins are (or, at least where) planned [planned](https://ice-and-fire-mod.fandom.com/wiki/Goblin). However, I _do_ think the respawn idea is nice. I definitely feel there should be more than 15. Also, I think they should be raid villages, _rareley_ killing villagers, sometimes kidnapping them (especially baby villagers), who they then force to work for them.

**Comment by NathanCPlaster** [comment]
What about different level of nests spawn which determines the number of goblins and how quickly they can respawn.  The kidnapping of villagers may be a little too much to create code for because at least with just this mod there is no carry system of mobs in place, at least in that way.  Could make it so that you can find various villagers in the nest and if you rescue them your trades are better.  Similar to a hero of the village status or when you cure a zombie villager.  As for raiding villages that could be a unique event where if you don't kill any of them within in a time frame and they are close to a village it could cause them to raid.  What do you think of this?

**Comment by CTH999** [comment]
yeah! I like that level thing! And, perhaps the level is based on that whole chief/ king. As they get "stronger" (Defeat more goblins, ect), they will start getting more goblins, perhaps even expanding!

Maybe the forest ones live in something similar to a village (but with walls), and the mountain ones live in caves (modified natural caves)

Also, perhaps they can kidnap villagers, but they rather "herd" the villagers, instead of carrying them? Or, spawning with them might be simpler, I was baseing that off of the "attacks villages" part

and, I love that unique event concept!

also, perhaps they will (rareley, and only at high levels/ if they kill a unicorn that they run into post-generation) have [Unicorn Horns](https://ice-and-fire-mod.fandom.com/wiki/Unicorn), which are said to be "powerful, but killing a unicorn brings a terrible curse". This could be a way of getting Unicorn Horns _without_ killing unicorns!

**Comment by NathanCPlaster** [comment]
They could use leads to take the villagers away for the capturing, and I like that idea of getting items from them base don interactions with other mobs!
